lib/transforms/bundleSystemJs.js

```diff
@@ -928,9 +939,14 @@ module.exports = function (options) {
             bundleStrategyOptions.conditions = options.conditions;
         }
 
+        var globalSnapshotBeforeBundling = Object.assign({}, global);
+        _.defaults(global, systemJsGlobals);
         return bundleStrategy(builder, entryPointNames.map(function (entryPointName) {
             return { name: entryPointName };
         }), bundleStrategyOptions).then(function (bundles) {
+            Object.keys(systemJsGlobals).forEach(function (key) {
+                delete global[systemJsGlobals];
```

Comment:
Whoops, that shouldn't even be there, the cleanup is handled in the `finally` block. Fixed.